test/metabase/query_processor_test/date_bucketing_test.clj

```diff
@@ -274,7 +274,7 @@
   ;; timezone
   ;;
   ;; TIMEZONE FIXME
-  (mt/test-drivers (mt/normal-drivers-except #{:h2 :sqlserver :redshift :sparksql :mongo :athena})
+  (mt/test-drivers (mt/normal-drivers-with-feature :test/jvm-timezone-setting)
```

Comment:
This excludes now `:vertica` too. It could be added back using `conj` which is ugly but easy, or we could introduce one more feature, but I have no good idea how that should be called.